addons/account/models/account_move.py

```diff
@@ -6210,6 +6210,22 @@ def _get_discount_allocation_account(self):
             return self.company_id.account_discount_income_allocation_id
         return None
 
+    def get_available_invoice_template_pdf_report_ids(self):
```

Comment:
good practice to put it as private also
_get_available_invoice_template_pdf_report_ids